test/ts/unit/collateralized/runners/seize_collateral.ts

```diff
@@ -0,0 +1,207 @@
+// modules
+import { expect } from "chai";
+import * as ABIDecoder from "abi-decoder";
+import * as moment from "moment";
+import { compact } from "lodash";
+import { BigNumber } from "bignumber.js";
+
+// wrappers
+import { MockCollateralizedTermsContractContract } from "types/generated/mock_collateralized_terms_contract";
+import { MockDebtRegistryContract } from "types/generated/mock_debt_registry";
+import { MockERC20TokenContract } from "types/generated/mock_e_r_c20_token";
+import { MockTokenRegistryContract } from "types/generated/mock_token_registry";
+
+// scenarios
+import { SeizeCollateralScenario, TestAccounts, TestContracts } from "./";
+
+// utils
+import { NULL_ADDRESS, REVERT_ERROR } from "../../../test_utils/constants";
+
+// logs
+import { CollateralSeized } from "../../../logs/collateralized_contract";
+
+export class SeizeCollateralRunner {
+    private contracts: TestContracts;
+    private accounts: TestAccounts;
+
+    constructor() {
+        this.testScenario = this.testScenario.bind(this);
+    }
+
+    public initialize(contracts: TestContracts, accounts: TestAccounts) {
```

Comment:
again, this feels like it could be easily handled by the constructor

Reply:
See above.